launchpad/launchpad_init.gno

```diff
@@ -90,6 +93,9 @@ func CreateProject(
 		panic("invalid start time, can not start project in past")
 	}
 
+	common.IsHalted()
+	en.MintAndDistributeGns()
```

Comment:
I see these two being added to all public functions. Just curious - does this have anything to do with the calculation issue in this PR? or are you just adding them?

Reply:
It DOES NOT affect calculation logic.
However IT DOES AFFECT test cases, because test cases does run other contract's too.